### DISCUSSION TITLE
Warn on empty search filter term?

### DISCUSSION THREAD

**Discussion by jay** [community/community#45496] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I wonder if I'm the only one doing this but occasionally I accidentally put a space between the [filter name](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests#about-search-terms) and value, like `author: foo`, when I search issues and that causes github to apparently search for foo without any author, which will never be found since all issues have authors.

**Comment by jge162** [comment]
A warning would be nice, like sometimes when you see alerts in coding. So that way you don't search query prematurely(with space).
